src/registrar/forms/application_wizard.py

```diff
@@ -549,13 +531,18 @@ class PurposeForm(RegistrarForm):
 
 
 class YourContactForm(RegistrarForm):
+    JOIN = "submitter"
+
     def to_database(self, obj):
         if not self.is_valid():
             return
         contact = getattr(obj, "submitter", None)
-        if contact is not None:
+        if contact is not None and not contact.has_more_than_one_join("submitted_applications"):
```

Comment:
(Q) Why is this using the related name "submitted_applications" instead of "submitter" (and vice/versa)